pages/admin/request/create-renewal.tsx

```diff
@@ -61,18 +76,203 @@ export default function CreateRenewal() {
+            phone: permitHolderInformation.phone,
+            email: permitHolderInformation.email,
+            rcdUserId: permitHolderRcdUserID,
+            updatedPhysician,
+            physicianName: doctorInformation.name,
+            physicianMspNumber: doctorInformation.mspNumber,
+            physicianAddressLine1: doctorInformation.addressLine1,
+            physicianAddressLine2: doctorInformation.addressLine2,
+            physicianCity: doctorInformation.city,
+            physicianPostalCode: doctorInformation.postalCode,
+            physicianPhone: doctorInformation.phone,
+            usesAccessibleConvertedVan: additionalQuestions.usesAccessibleConvertedVan,
+            requiresWiderParkingSpace: additionalQuestions.requiresWiderParkingSpace,
+            shippingAddressSameAsHomeAddress: paymentDetails.shippingAddressSameAsHomeAddress,
+            billingAddressSameAsHomeAddress: paymentDetails.billingAddressSameAsHomeAddress,
+            ...(paymentDetails.shippingAddressSameAsHomeAddress === false && {
+              shippingFullName: paymentDetails.shippingFullName,
+              shippingAddressLine1: paymentDetails.shippingAddressLine1,
+              shippingAddressLine2: paymentDetails.shippingAddressLine2,
+              shippingCity: paymentDetails.shippingCity,
+              shippingProvince: paymentDetails.shippingProvince,
+              shippingPostalCode: paymentDetails.shippingPostalCode,
+            }),
+            ...(paymentDetails.billingAddressSameAsHomeAddress === false && {
+              billingFullName: paymentDetails.billingFullName,
+              billingAddressLine1: paymentDetails.billingAddressLine1,
+              billingAddressLine2: paymentDetails.billingAddressLine2,
+              billingCity: paymentDetails.billingCity,
+              billingProvince: paymentDetails.billingProvince,
+              billingPostalCode: paymentDetails.billingPostalCode,
+            }),
+            donationAmount: paymentDetails.donationAmount,
+            paymentMethod: paymentDetails.paymentMethod,
+          },
+        },
+      });
+    }
+  };
+
   return (
     <Layout>
       <GridItem display="flex" flexDirection="column" colSpan={12} paddingX="108px">
         <Flex>
           <Text textStyle="display-large">
-            New Renewal Request (User ID:{' '}
-            <Box as="span" color="primary">
-              <Link href={`/permit-holder/${permitHolderID}`}>
-                <a>{permitHolderID}</a>
-              </Link>
-            </Box>
-            )
+            {`New Renewal Request`}
+            {permitHolderRcdUserID && ` (User ID: `}
+            {permitHolderRcdUserID && (
+              <Box as="span" color="primary">
+                <Link href={`/permit-holder/${permitHolderRcdUserID}`}>
+                  <a>{permitHolderRcdUserID}</a>
+                </Link>
+              </Box>
+            )}
+            {permitHolderRcdUserID && `)`}
```

Comment:
To avoid using the conditional 3 times, we can wrap the 3 parts using a fragment (`<></>`)